### PR TITLE
Update chainlink interface to V3

### PR DESCRIPTION
**Pull request by haythem96**
# Task: Update chainlink interface to V3

## High Level Description

This PR update Chainlink interface from V2 to V3.

### Code

- [x] Unit test 100% coverage
- [x] Does your code follow the naming and code documentation guidelines?

### Documentation

- [x] Is your code up to date with the spec? 
- [x] Have you added your tests to the testing doc?

### CODE REVIEW

File: contracts/interfaces/AggregatorInterface.sol [hunk 1]
```diff
@@ -7,16 +7,34 @@ pragma solidity 0.6.10;
...
-    event NewRound(uint256 indexed roundId, address indexed startedBy, uint256 startedAt);
+    function latestRoundData()
+        external
+        view
+        returns (
+            uint80 roundId,
+            int256 answer,
+            uint256 startedAt,
+            uint256 updatedAt,
+            uint80 answeredInRound
```
Comment (aparnakr) [inline contracts/interfaces/AggregatorInterface.sol:38]: what is the answered in round vs the roundId?

Comment (aparnakr) [reply contracts/interfaces/AggregatorInterface.sol:38]: Do we need some check around that?

**Review by aparnakr — DISMISSED** [review]

**Review by antoncoding — CHANGES_REQUESTED** [review]
In their v3, [the aggregator also return `decimals`. ](https://docs.chain.link/docs/price-feeds-api-reference#decimals)we should use that instead of assuming things will be 8 decimals

**Comment by haythem96** [comment]
> In their v3, [the aggregator also return `decimals`. ](https://docs.chain.link/docs/price-feeds-api-reference#decimals)we should use that instead of assuming things will be 8 decimals

Oh cool, added that.

**Review by antoncoding — APPROVED** [review]

**Review by aparnakr — APPROVED** [review]

File: contracts/interfaces/AggregatorInterface.sol (hunk 1 above)
Comment (haythem96) [reply contracts/interfaces/AggregatorInterface.sol:38]: Will look into that

**Pull request merged**